internal/pkg/cli/env_delete.go

```diff
@@ -169,9 +172,23 @@ func (o *deleteEnvOpts) Execute() error {
 		return err
 	}
 	o.prog.Stop(log.Ssuccessf(fmtRetainEnvRolesComplete, o.name))
+
 	if err := o.deleteStack(); err != nil {
+		o.prog.Stop(log.Serrorf("Failed to delete resources for the %q environment\n", o.name))
+		return err
+	}
+
+	envInfo, err := o.getEnvConfig()
+	if err != nil {
+		return err
+	}
+	// Un-delegate DNS and optionally delete stackset instance.
+	o.prog.Start("Cleaning up app-level resources and permissions\n")
+	if err := o.cleanUpAppResources(); err != nil {
+		o.prog.Stop(log.Serrorf("Failed to remove ECR Repos and S3 bucket from region %s\n", envInfo.Region))
```

Comment:
```suggestion
		o.prog.Stop(log.Serrorf("Failed to remove environment resource from app"))
```

If I understand it correctly, we don't remove ECR repos and S3 buckets from region, if the environment is _not_ the last environment in that region. Therefore, I feel like we should just print a more general log here.